### ISSUE TITLE
Compatibility with the Google Reader API

### ISSUE THREAD

**Issue by robinmetral** [omnivore-app/omnivore#3984]
Has Omnivore considered adding a compatibility layer for the Google Reader API? This would make a subset of Omnivore's features compatible with [many native apps](https://github.com/FreshRSS/FreshRSS?tab=readme-ov-file#apis--native-apps).

This would support only a subset of Omnivore's features around consuming feeds (excluding highlighting, for instance). Omnivore users could download and read new entries in the feeds that they subscribe to, using native clients.

Related: #1862

**Comment by jacksonh** [comment]
Thanks, is this more to get data into Omnivore or to have native clients for reading?

**Comment by robinmetral** [comment]
Both: reading on native clients, but the read status at least should make it back to Omnivore. Ideally also adding a subscription on a native client would add it to Omnivore.

Don't take this as a formal feature request — it's probably a lot of work and I don't know if other users would need anything like this. A compat layer for the API might just add unnecessary bloat. I was mostly wondering whether this is something that had been discussed/considered, if it's on any kind of roadmap, etc.

Thanks!

**Comment by jacksonh** [comment]
yeah honestly I hadn't even heard of this API so I do want to research it more. But also wondering why you'd use Omnivore if you like another client better? I'm guessing something like want to us native desktop but still use our native mobile apps?

**Comment by robinmetral** [comment]
> I'm guessing something like want to us native desktop but still use our native mobile apps?

Yup native desktop would be one reason. Helpful especially for offline use. I could also imagine people who like other clients better, but want to use Omnivore as the backend RSS aggregator (instead of e.g. FreshRSS, TTRSS, or the like).

> I hadn't even heard of this API

That's fair, Google Reader has been retired years ago :sweat_smile: its API just somehow remained sort of a "standard" in the RSS ecosystem, so a bunch of aggregators and readers have support for it, for interop with other tools.

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
